File: library/src/main/java/com/example/library/dto/response/BookResponse.java
package com.example.library.dto.response;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BookResponse {

    private Integer id;

    private String bookCode;
    
    private String bookName;

    private Integer price;

    private String categoryName;

    private String createdBy;
        
    private Integer categoryId;
    
    private LocalDateTime updatedAt;
    
    private LocalDateTime deleteAt;

}

File: MyApplication/app/src/main/java/com/example/myapplication/data/remote/dto/response/BookResponse.kt
package com.example.myapplication.data.remote.dto.response

data class BookResponse(
    val id: Int,
    val bookCode: String,
    val bookName: String,
    val price: Int,
    val categoryName: String,
    val createdBy: String,
    val accountId : Int,
    val categoryId : Int
)